contact/Makefile

```diff
@@ -0,0 +1,12 @@
+# Wrap TTL files into JS files for bundling with library
+
+,all : individualForm.js vcard.js
+
+individualForm.js : individualForm.ttl
+				(echo 'module.exports = `' ; cat individualForm.ttl; echo '`') >  individualForm.js
```

Comment:
So this is one way of exporting arbitrary content from within JS modules. Another way would be to use a tool like `webpack` and to design a custom [loader](https://webpack.js.org/concepts/loaders/) for turtle. Perhaps one even exists.